frontstage/templates/sign-in/sign-in.html

```diff
@@ -39,6 +39,17 @@ <h1 class="panel__title venus">{{ errorType[error][0] }}</h1>
         <h1 class="saturn">You've activated your account</h1>
         <p>You may now sign in.</p>
         {% else %}
+
+        {% if config.AVAILABILITY_BANNER %}
+        <div class="panel panel--simple panel--error">
```

Comment:
not sure we should be using an error styling for this message, shouldn't it be an information panel? see the sdc-pattern-library https://onsdigital.github.io/sdc-global-design-patterns/components/detail/panel.html

Reply:
Cool, im going to have a discussion with Nic this morning